public/pages/CreateTrigger/components/Action/actions/Message.js

```diff
@@ -194,6 +195,8 @@ export default function Message(
     _.set(values, actionableAlertsSelectionsPath, actionableAlertsSelections);
   }
 
+  if (!displayThrottlingSettings) _.set(values, `${actionPath}.throttle_enabled`, false);
```

Comment:
Is `_.set(values, `${actionPath}.throttle_enabled`, false);` set to true by default? or should we set it to true here in the else case?

Reply:
By default, `throttle_enabled` is [false](https://github.com/opensearch-project/alerting-dashboards-plugin/blob/main/public/pages/CreateTrigger/utils/constants.js#L49). 

Setting `throttle_enabled` to false here helps ensure throttling settings aren't sent to the backend for action configurations that don't support throttling when the user may have previously been experimenting with configurations that do support such settings.

Reply:
Aah, so if the user was playing with the setting for PER_ALERT and then goes to PER_EXECUTION then we should omit it in that case, right?

Reply:
For bucket level monitors, we want to offer `PER_ALERT` actions, but we do not want to offer them for `PER_EXECUTION`; however, the bug prevented throttling configurations for both selections.